posts/inside-rust/2020-04-23-Governance-wg.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Governance Working Group Update: Meeting 23 April 2020"
+team: The Governance WG <https://github.com/rust-lang/wg-governance>
```

Comment:
It's missing the author field. 😄

Reply:
Is that really needed?

Reply:
Currently yes it's required in the software. If you don't want to sign your name the team field is actually optional so you could just change that to `author`.
```suggestion
author: The Governance WG <https://github.com/rust-lang/wg-governance>
```